python-lib/dku_deeplearning_image/tensorboard_handler.py

```diff
@@ -35,34 +48,20 @@ def __get_custom_assets_zip_provider():
 
 def init_flags(loader_list):
     parser = ArgumentParser()
-    map(lambda x: x.define_flags(parser), loader_list)
+    for x in loader_list:
+        x.define_flags(parser)
     flags = parser.parse_args([])
     return flags
 
 
-def make_plugin_loader(plugin_spec):
-    """Returns a plugin loader for the given plugin.
-    Args:
-      plugin_spec: A TBPlugin subclass, or a TBLoader instance or subclass.
-    Returns:
-      A TBLoader for the given plugin.
-    """
-    if issubclass(plugin_spec, base_plugin.TBLoader):
-        return plugin_spec()
-    if issubclass(plugin_spec, base_plugin.TBPlugin):
-        return base_plugin.BasicLoader(plugin_spec)
-    raise TypeError("Not a TBLoader or TBPlugin subclass: %r" % (plugin_spec,))
-
-
 def __get_tb_app(tensorboard_logs):
     plugins_or_loaders = tb_default.get_plugins()
```

Comment:
now that `make_plugin_loader` is not called and removed, this variable can be renamed to plugins, no?
btw, why did we remove `make_plugin_loader`?

Reply:
It seems that `standard_tensorboard_wsgi` accepts both plugins and loaders, so I do not need to transform the plugins into loaders. Do you remember why you wrote it in the first place in dip? Didn't `standard_tensorboard_wsgi` supported plugins?

Reply:
nope, tbh don't remember why I had to do it. Well as long as it works and the name of the variable is not misleading anymore it's fine for me